src/main/java/net/mcreator/workspace/elements/FolderElement.java

```diff
@@ -47,7 +49,7 @@ public void updateStructure() {
 		});
 	}
 
-	public void addChild(FolderElement child) {
+	public @Nullable FolderElement addChild(FolderElement child) {

```

Comment:
Why did you add this?

Reply:
Things were needed or else it wouldn't compile
(just did things Intellij told me to do for it to compile)

Reply:
Please restructure the code the way this will not be needed as I don't see a real need for this.

Reply:
I get this if i don't make it public `@Nullable FolderElement addChild(FolderElement child) `
![image](https://user-images.githubusercontent.com/60476871/116905439-08cdd380-ac50-11eb-90ed-ea97e6d241e9.png)

Reply:
Split this into two statements and it will work. Also please don't add new things for things already implemented

Reply:
Have a look on Discord for something